reascripts/ReaSpeech/source/ASRControls.lua

```diff
@@ -10,6 +10,11 @@ ASRControls = PluginControls {
   DEFAULT_LANGUAGE = '',
   DEFAULT_MODEL_NAME = 'small',
 
+  HELP_MODEL = 'Model to use for transcription. Larger models provide better accuracy but use more resources.',
+  HELP_LANGUAGE = 'Language spoken in audio. Set to "Detect" to auto-detect language.',
+  HELP_PRESERVED_WORDS = 'Comma-separated list of words to preserve in transcript. Example: Jane Doe, CyberCorp',
+  HELP_VAD = 'Enable Voice Activity Detection (VAD) to filter out silence.',
```

Comment:
i could see this as potentially confusing based on the rabbit hole i just fell into thinking about it: "wait, so if this is off does it...transcribe silence?" lol

Reply:
I wasn't sure whether to explicitly mention "hallucinations", but indeed it does try to transcribe silence, and the result is often nonsense or text like "Thanks for watching!"

Reply:
alternate solution: checkbox that adds "thanks for watching" to all transcripts